pkg/sbom/collectors/host/request.go

```diff
@@ -27,18 +19,18 @@ type scanRequest struct {
 }
 
 // NewScanRequest creates a new scan request
-func NewScanRequest(path string, fs fs.FS) sbom.ScanRequest {
+func NewScanRequest(path string, fs fs.FS) types.ScanRequest {
 	return scanRequest{Path: path, FS: fs}
 }
 
 // Collector returns the collector name
 func (r scanRequest) Collector() string {
-	return collectors.HostCollector
+	return "host"
 }
 
 // Type returns the scan request type
-func (r scanRequest) Type(sbom.ScanOptions) string {
-	return sbom.ScanFilesystemType
+func (r scanRequest) Type(types.ScanOptions) string {
+	return "filesystem"
```

Comment:
Should we use here the constant defined in `types.go` ?

```suggestion
	return types.ScanFilesystemType
```